src/main/java/foodwhere/commons/core/Messages.java

```diff
@@ -6,9 +6,10 @@
 public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
-    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! %1$s";
     public static final String MESSAGE_INVALID_STALL_DISPLAYED_INDEX = "The stall index provided is invalid";
     public static final String MESSAGE_INVALID_REVIEW_DISPLAYED_INDEX = "The review index provided is invalid";
+    public static final String MESSAGE_INVALID_INDEX = "Index is not a single non-zero unsigned integer.\n";
```

Comment:
This seems a bit too technical for the user, talking about "unsigned integer" - how about "The index cannot be read as a positive integer."?